### ISSUE TITLE
Package unavailable for download wazuhapp-3.10.2_7.4.0.zip 

### ISSUE THREAD

**Issue by jpuris** [wazuh/wazuh-kibana-app#1855]
| Wazuh | Elastic | Rev |
| ----- | ------- | --- |
| 3.10.2   | 7.4.0     |  |

**Description**
The package `wazuhapp-3.10.2_7.4.0.zip` is unavailable for download. It was downloadable couple days ago, however.

**Steps to reproduce**

1. Go to https://packages.wazuh.com/wazuhapp/wazuhapp-3.10.2_7.4.0.zip
2. Access denied, while https://packages.wazuh.com/wazuhapp/wazuhapp-3.10.2_7.3.2.zip works

**Additional context**
```# sudo -u kibana /usr/share/kibana/bin/kibana-plugin install https://packages.wazuh.com/wazuhapp/wazuhapp-3.10.2_7.4.0.zip
Attempting to transfer from https://packages.wazuh.com/wazuhapp/wazuhapp-3.10.2_7.4.0.zip
Attempting to transfer from https://artifacts.elastic.co/downloads/kibana-plugins/https://packages.wazuh.com/wazuhapp/wazuhapp-3.10.2_7.4.0.zip/https://packages.wazuh.com/wazuhapp/wazuhapp-3.10.2_7.4.0.zip-7.4.0.zip
Plugin installation was unsuccessful due to error "No valid url specified."```

**Comment by adri9valle** [comment]
Hi @jpuris,

Currently, we're still working on Wazuh 3.11 which will be compatible with Elasticsearch / Kibana 7.4.

We'll notified you when Wazuh 3.11 was released, until that moment the matrix compatibility is the following:

| Kibana version | Wazuh app version | Installation                                                                                                |
| :------------: | :---------------: | :---------------------------------------------------------------------------------------------------------- |
|      6.8.2     |       3.10.1      | /usr/share/kibana/bin/kibana-plugin install <https://packages.wazuh.com/wazuhapp/wazuhapp-3.10.1_6.8.2.zip> |
|      7.3.1     |       3.10.1      | /usr/share/kibana/bin/kibana-plugin install <https://packages.wazuh.com/wazuhapp/wazuhapp-3.10.1_7.3.1.zip> |
|      7.3.2     |       3.10.1      | /usr/share/kibana/bin/kibana-plugin install <https://packages.wazuh.com/wazuhapp/wazuhapp-3.10.1_7.3.2.zip> |
|      6.8.3     |       3.10.2      | /usr/share/kibana/bin/kibana-plugin install <https://packages.wazuh.com/wazuhapp/wazuhapp-3.10.2_6.8.3.zip> |
|      7.3.2     |       3.10.2      | /usr/share/kibana/bin/kibana-plugin install <https://packages.wazuh.com/wazuhapp/wazuhapp-3.10.2_7.3.2.zip> |

Wazuh 3.11 will be also released with Elasticsearch and Kibana 6.8.3 and 7.2.0.

Regards,

**Comment by jpuris** [comment]
Thanks for the info. Somehow I had managed to download and install 3.10.2_7.4.0 on some machines last week, hence the question. Go figure.

Will wait for release of 3.11 then.
